x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -268,3 +273,7 @@ enrichCommand
 enrichWithClause
     : (newName=qualifiedNamePattern ASSIGN)? enrichField=qualifiedNamePattern
     ;
+
+lookupCommand
+    : LOOKUP tableName=qualifiedNamePattern ON matchFields=qualifiedNamePatterns
```

Comment:
I'm really not sure if this should use a qualified name pattern for the table name. IMHO the table name should be somewhat in line with `FROM`'s index name and `ENRICH`'s policy name. Qualified name patterns are usually used for field names, so we can do stuff like
```
KEEP `some-thing`.`something-else`.yet_something_else
```

`FROM` uses an `INDEX_UNQUOTED_IDENTIFIER`, `ENRICH` a similar `ENRICH_POLICY_NAME_BODY`.

Maybe we just align with `FROM` here and use `INDEX_UNQUOTED_IDENTIFIER` as well, unless table names need to be more general than that?

Reply:
I'll give it a go.